fastboot-tests/fixtures/treeshake/ember-cli-build.js

```diff
@@ -0,0 +1,11 @@
+const EmberApp = require('ember-cli/lib/broccoli/ember-app');
+
+module.exports = function(defaults) {
+  let app = new EmberApp(defaults, {
+    'ember-bootstrap': {
+      whitelist: ['bs-button-group']
```

Comment:
Sounds good. Maybe we want to have an `all` alias that we could default to. I'm fine going without the blueprint. I'll create an issue in both projects to address it.